textalyze.js

```diff
@@ -1,7 +1,16 @@
+/**
+ * Given an input String, returns true if it's a valid string false otherwise
+ * @param {String} string - The String to be sanitized
```

Comment:
I love that you're adding your own JSDoc and following the conventions set in the code.  The description of the first parameter is wrong though — you forgot to change it after copying and pasting. :)